nextflow_schema.json

```diff
@@ -140,6 +140,30 @@
                 }
             }
         },
+        "viral_contig_extension": {
+            "title": "Viral contig extension",
+            "type": "object",
+            "description": "Extend viral contigs",
+            "default": "",
+            "properties": {
+                "run_cobra": {
+                    "type": "boolean",
+                    "description": "Run COBRA to extend viral contigs"
+                },
+                "cobra_assembler": {
+                    "type": "string",
+                    "description": "The assembler that was used to assemble viral contigs"
+                },
+                "cobra_mink": {
+                    "type": "string",
+                    "description": "Minimum kmer value used during assembly"
+                },
+                "cobra_maxk": {
+                    "type": "string",
+                    "description": "Maximum kmer value used during assembly"
```

Comment:
Rather sparse, or are you doing a docs pass prior release?

Reply:
Yes, I'm planning to add a lot more detail once the pipeline is in it's final form!